packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -137,6 +137,13 @@ class Ticker {
   /// [isTicking].
   bool get isActive => _future != null;
 
+  /// The start time of the ticker.
```

Comment:
The Ticker is designed to measure relative time. I don't see a good reason to change that.

Reply:
Well it is required for flutter_smooth, mainly here: https://github.com/fzyzcjy/flutter_smooth/blob/0c5db0ff270aa0c8cff28ea19055999627a8df6d/packages/smooth/lib/src/drop_in/list_view/shift.dart#L352-L356

Shortly speaking, I need to know the time that the ListView animation is using, such that I can fully replicate it in an external extra SmoothShift.